spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteTablePathsAction.java

```diff
@@ -386,22 +408,23 @@ public void testDeleteDataFile() throws Exception {
         .hasSize(1);
   }
 
-  @Test
+  @TestTemplate
   public void testPositionDeletesParquet() throws Exception {
     runPositionDeletesTest("parquet");
   }
 
-  @Test
+  @TestTemplate
   public void testPositionDeletesAvro() throws Exception {
     runPositionDeletesTest("avro");
   }
 
-  @Test
+  @TestTemplate
   public void testPositionDeletesOrc() throws Exception {
     runPositionDeletesTest("orc");
   }
 
   private void runPositionDeletesTest(String fileFormat) throws Exception {
+    assumeThat(formatVersion).isEqualTo(2);
```

Comment:
we shouldn't be skipping this for v3. Instead we need to pass the formatVersion further below so that the right type of delete file is created